gammapy/modeling/models/spatial.py

```diff
@@ -507,7 +507,7 @@ def to_dict(self):
     @staticmethod
     def evaluate(lon, lat, value):
         """Evaluate model."""
-        return value
+        return u.Quantity(value, "sr-1", copy=False)
```

Comment:
Same here, I think there is no gain in forcing the output unit here. The output unit depends on the parameter unit of the model, which is just fine. I users want to have `sr-1` they can always do `model(lon, lat).to("sr-1")`. Instead it comes with a minimal performance (not really relevant) drawback, because we transform the units and make a copy if it is different.